Proposal for SECURITY.md (#231)
uptane/uptane-standard: SECURITY.md

# Security Policy

> The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata in the Uptane Standard, please follow these guidelines for responsible disclosure.

Reviewer: ```suggestion
The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata—including security issues—in the Uptane Standard, please follow these guidelines for responsible disclosure.
```

Reviewer 2: This is fine with me. @trishankatdatadog thanks for the assist. Can you merge? It's been reviewed multiple times

# Security Policy

Think you've found a vulnerability in the Uptane Standard? Please review the following details for responsible disclosure.

## Supported Versions

We release updates to the Uptane specification to address security vulnerabilities. Which versions are eligible to receive updates are listed below:

> | Version | Supported          |

Author: Should we support all released versions?

Reviewer 3: What does "support" mean in this case? IMO it doesn't really make sense to update released versions. We could conceivably patch older releases with bugfixes, but I'd probably rather just focus on updating the latest version.

Reviewer 2: I'm inclined to agree with @pattivacek here that we should fix only the current version, but does this raise any potential backwards compatibility issues? I can't think of any changes to date that would break compatibility, but might it be an issue in the future?

Reviewer 4: I agree, we can probably just say "you may report errata for the most recent standard version" or something.

| ------- | ------------------ |
| 1.2.0   | :white_check_mark: |
| 1.1.0   | :white_check_mark: |
| 1.0.1   | :white_check_mark: |
| 1.0.0   | :white_check_mark: |

## Reporting a Vulnerability

> Please report (suspected) security vulnerabilities to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.

Reviewer 3: Who is behind that email address? This can be discussed elsewhere if we don't want that to be publicly disclosed right now. It's just the first I've heard of that and I'm curious who is taking that responsibility on.

Reviewer 2: The question of where these reports go is a good one. I don't mind monitoring and referring these reports, but ideally whoever gets those responses should be able to address the problem. I think its worth bringing this up in one of our meetings. I'm listed as a maintainer for the projects on the website, but it would be good to have a technical person listed as well to address security flaws.

> Please report (suspected) security vulnerabilities to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.

Reviewer 2: Thanks for the comments @trishankatdatadog . I've opened a PURE to deal with this issue. It's not a problem to leave comments here, but it might be better going forward to post comments at https://github.com/uptane/pures/pull/8 .

# Security Policy

Think you've found an errata in the Uptane Standard? Please review the following details for responsible disclosure.

## Supported Versions

> We release updates to the Uptane specification to address errata. Which versions are eligible to receive updates are listed below:

Reviewer 2: ```suggestion
We release updates to the Uptane specification to address errata. The versions eligible to receive updates are listed below:
```

| Version | Supported          |
| ------- | ------------------ |
| 2.0.0   | :white_check_mark: |
| 1.2.0   | :white_check_mark: |
| 1.1.0   | :white_check_mark: |
| 1.0.1   | :white_check_mark: |
| 1.0.0   | :white_check_mark: |

## Reporting an Errata

> Please report (suspected) errata in the specification to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.

Reviewer 2: ```suggestion
Please report (suspected) errata in the specification to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind that Uptane is a community based project of volunteers and, therefore, response times could vary. If the issue is confirmed, we will provide a response within 90 days.
```

## Guidelines

We're committed to working with security researchers to resolve the erratas they discover. You can help us by following these guidelines:

*   Report errata to our dedicated security email (see [Reporting an Errata](#reporting-an-errata))
*   Please give as much detail as possible for a suspected errata in Uptane including:
     *  Version found
     *  Description of errata
     *  Examples (if applicable)

> *   Be patient - Give us a reasonable time to correct the issue before you disclose the errata. We care deeply about security, however we're an open-source project and our team is mostly comprised of volunteers.

Reviewer 2: Do we have to repeat this message ("Be patient, etc....") here?

# Security Policy

The Uptane community is committed to maintaining a reliable and consistent Standard. If you believe you have identified errata in the Uptane Standard, please follow these guidelines for responsible disclosure.

## Supported Versions

We release updates to the Uptane specification to address errata. The versions eligible to receive updates are listed below:

| Version | Supported          |
| ------- | ------------------ |
| 2.0.0   | :white_check_mark: |
| 1.2.0   | :white_check_mark: |
| 1.1.0   | :white_check_mark: |
| 1.0.1   | :white_check_mark: |
| 1.0.0   | :white_check_mark: |

## Reporting Errata

> Please report (suspected) errata in the specification to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind that Uptane is a community based project of volunteers and, therefore, response times could vary. If the issue is confirmed, we will provide a response within 90 days.

Reviewer 4: Do we need a separate mailing list for this rather than just using the GitHub issue tracker? We decided not to make a private process, so we can probably use the existing uptane-standards mailing list if the issue tracker is not sufficient

## Supported Versions

We release updates to the Uptane specification to address errata. You may report errata for the most recent version of the Uptane Standard. We will not retroactively make changes to older versions.

## Reporting Errata

Please report (suspected) errata in the specification. You can create an issue in the appropriate repository or send feedback directly to our mailing list at uptane-standards [at] googlegroups [dot] com.

## Guidelines

We're committed to working with security researchers to resolve errata they discover. You can help us by following these guidelines:

*   Please give as much detail as possible for a suspected errata in Uptane including:

>      *  Version found

Reviewer 2: ```suggestion
     *  Version in which it was found
```

| 1.0.1   | :white_check_mark: |
| 1.0.0   | :white_check_mark: |

> ## Reporting an Errata

Reviewer 3: Sorry to be annoying, but the singular is "erratum". Maybe it'd better to just write "Reporting errata" here?

Please report (suspected) errata in the specification to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.

## Guidelines

> We're committed to working with security researchers to resolve the erratas they discover. You can help us by following these guidelines:

Reviewer 3: "to resolve errata they discover"?
